dependency.gradle

```diff
@@ -25,17 +46,44 @@ ext.versions = [
 ext.library = [
         // Android dependencies
         androidSupportAppCompat: "com.android.support:appcompat-v7:$versions.androidSupportLibrary",
+        androidSupportDesign: "com.android.support:design:$versions.androidSupportLibrary",
+        androidSupportRecyclerView: "com.android.support:recyclerview-v7:$versions.androidSupportLibrary",
 
         dagger: "com.google.dagger:dagger:$versions.dagger",
         daggerCompiler: "com.google.dagger:dagger-compiler:$versions.dagger",
 
         rxJava: "io.reactivex.rxjava2:rxjava:$versions.rxJava",
         rxLint: "nl.littlerobots.rxlint:rxlint:$versions.rxLint",
 
+        autoValue: "com.google.auto.value:auto-value:$versions.autoValue",
+
         // Test dependencies
```

Comment:
Dependencies need re-ordering a bit, there are app based dependencies in the test area.
Perhaps `App dependencies` would make a better section heading too?